modules/about-must-gather.adoc

```diff
@@ -23,32 +23,31 @@ Alternatively, you can collect specific information by running the command with
 
 * To collect data related to one or more specific features, use the `--image` argument with an image, as listed in a following section.
 +
-For example:
-+
+
+.Example command
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather \
-  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.9.0
 ----
 
 * To collect the audit logs, use the `-- /usr/bin/gather_audit_logs` argument, as described in a following section.
 +
-For example:
-+
+
+.Example command
 [source,terminal]
 ----
 $ oc adm must-gather -- /usr/bin/gather_audit_logs
 ----
-+
```

Comment:
Just curious, are you intentionally removing the indentation of this note from being under the above list item? It seems like it has to do with the audit logs mentioned in bullet point # 2